lib/src/component_declaration/component_base.dart

```diff
@@ -91,6 +93,10 @@ ReactDartComponentFactoryProxy registerAbstractComponent(Type abstractComponentC
 /// via a fluent-style builder interface.
 typedef TProps UiFactory<TProps extends UiProps>([Map backingProps]);
 
+extension UiFactoryHelpers on UiFactory {
+  UiFactoryConfig<TProps> asForwardRefConfig<TProps extends bh.UiProps>({String displayName}) => UiFactoryConfig(propsFactory: PropsFactory.fromUiFactory(this), displayName: displayName);
```

Comment:
This should have a doc comment (maybe just a simple description and just point to `uiForwardRef`)